account_banking_pain_base/models/account_payment_mode.py

```diff
@@ -27,3 +27,8 @@ class AccountPaymentMode(models.Model):
         "- Country code (2, optional)\n"
         "- Company idenfier (N, VAT)\n"
         "- Service suffix (N, issued by bank)")
+    initiating_party_schema = fields.Char(
+        string='Initiating Party Schema', size=35,
+        help="This will be used as the 'Initiating Party Schema Name' in "
+        "the PAIN files generated by Odoo. If not defined, no schema will "
+        "be used.\n")
```

Comment:
It would be nice to have some more help (such as example values, and possibly a pointer to the specification).